src/app/components/client/client.controller.js

```diff
@@ -22,6 +22,8 @@
       var error = function (err) {
         $exceptionHandler(error, 'Save client');
       };
+      $scope.client.redirectUrls = $scope.client.redirectUrls.split("\n");
```

Comment:
Maybe it would be good to note within the UI that they should be separated by line breaks and not something else like a comma.